gcp-intro/python/README.md

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
Please only put the same workshop in once. There's no need for it to be in two places that we need to maintain.

Reply:
Not intentional! Removed.